include/stl2/view/view_interface.hpp

```diff
@@ -94,14 +94,14 @@ STL2_OPEN_NAMESPACE {
 			constexpr decltype(auto) at(difference_type_t<iterator_t<R>> n) {
 				return n < 0 || n >= __stl2::size(derived())
 					? throw std::out_of_range{}
-					: (*this)[n];
+					: derived()[n];
```

Comment:
Why?

Reply:
The derived type might implement `operator[]`.